docs/topics/proc_installing-launcher-operator.adoc

```diff
@@ -126,7 +126,7 @@ Based on the templates, OpenShift automatically installs:
 * an access role definition for the Launcher operator
 * access role bindings for the Launcher operator
 
-. Install the `launcher_cr.yaml` custom resource to initiate the deployment of the {name-launcher} tool.
+. Customize the `launcher_cr.yaml` custom resource with your OpenShift Console URL and install it in order to initiate the deployment of the {name-launcher} tool.
```

Comment:
I think this works better as two separate steps: modify the custom resource and then install it. Something like:

8. Edit the 'launcher_cr.yml' custom resource to add your OpenShift Console URL:

####### OpenShift Configuration
  openshift:
    consoleUrl: #<the OpenShift Console Url (ex: 'https://192.168.64.4:8443')>#

9. Install the 'launcher_cy.yaml' custom resource to deploy the {name-launcher} tool.

Reply:
I do agree with Huntley here. I'd like a bit more two steps approach with an example snippet.